community/consistency-check/src/test/java/org/neo4j/consistency/checking/full/FullCheckIntegrationTest.java

```diff
@@ -556,11 +556,13 @@ public void shouldReportNodesThatAreNotIndexed() throws Exception
         while ( indexRuleIterator.hasNext() )
         {
             IndexRule indexRule = indexRuleIterator.next();
-            IndexAccessor accessor = fixture.directStoreAccess().indexes().getOnlineAccessor(
-                    indexRule.getId(), indexRule.getIndexDescriptor(), samplingConfig );
+            IndexAccessor accessor = fixture.directStoreAccess().indexes().
+                    apply( indexRule.getProviderDescriptor() ).getOnlineAccessor(
+                            indexRule.getId(), indexRule.getIndexDescriptor(), samplingConfig );
             IndexUpdater updater = accessor.newUpdater( IndexUpdateMode.ONLINE );
             updater.remove( asPrimitiveLongSet( indexedNodes ) );
             updater.close();
+            accessor.force();
```

Comment:
That's the general contract and something that native indexing hooks in on to checkpoint properly. It's just that lucene happened to make do without it.